### PR TITLE
Feature/initial dom manipulation

### PR DESCRIPTION
**Pull request by tylrs**

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -1,7 +1,81 @@
...
+function convertUserInfo(userSearch) {
+  let convertedUserSearch = {
+    type: 'name',
+    query: [`${userSearch}`]
+  }
+  return convertedUserSearch;
+}
+
+// other functions
+renderRecipes(recipeRepository.recipes);
```
Comment (andrewvallejo) [inline src/scripts.js:46]: I would add this to the window loads event listener

File: src/scripts.js [hunk 2]
```diff
@@ -1,7 +1,81 @@
...
+//rendering tags....
+//loop through recipe tags
+
+function renderRecipes(recipes) {
```
Comment (andrewvallejo) [inline src/scripts.js:50]: This is clean! Can't wait to see it in action!

File: src/scripts.js [hunk 3]
```diff
@@ -1,7 +1,81 @@
...
+// event handlers
+function collectUserInfo() {
+  let userSearch = searchBar.value;
+  let convertedUserSearch = convertUserInfo(userSearch)
+  let filteredRecipes = recipeRepository.filterRecipes(convertedUserSearch);
+  renderRecipes(filteredRecipes)
+  console.log(filteredRecipes);
+}
+
+function convertUserInfo(userSearch) {
```
Comment (andrewvallejo) [inline src/scripts.js:37]: I would love to see this be dynamic

File: src/scripts.js (hunk 1 above)
Comment (tylrs) [reply src/scripts.js:46]: Ooh you're right, I will do that

**Review by andrewvallejo — APPROVED** [review]

**Pull request merged**